### PR TITLE
CNFT1-1887 moved advanced search from pages to apps module

### PR DESCRIPTION
**Pull request by mdavis-enquizit**
## Description

CNFT1-1887 moved advanced search from pages to apps module

## Tickets

* [CNFT1-1887](https://cdc-nbs.atlassian.net/browse/CNFT1-1887)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [ ] Code has been fully tested to meet acceptance criteria
- [ ] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [ ] All new functions/classes/components reasonably small
- [ ] Functions/classes/components focused on one responsibility
- [ ] Code easy to understand and modify (clarity over concise/clever)
- [ ] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [ ] PR does not contain hardcoded values (Uses constants)
- [ ] All code is covered by unit or feature tests

[CNFT1-1887]: https://cdc-nbs.atlassian.net/browse/CNFT1-1887?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/pages/advancedSearch/components/LabReportResults.tsx
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/pages/advancedSearch/components/LabReportResults.tsx:1]: This file should be in `apps/modernization-ui/src/apps/labReportSearch`

**Review by adamloup-enquizit — CHANGES_REQUESTED** [review]
Please move the `LabReportResults.tsx` file to `apps/modernization-ui/src/apps/labReportSearch`

File: apps/modernization-ui/src/apps/search/event/components/labReportSearch/LabReportResults.spec.tsx [hunk 1]
```diff
@@ -87,13 +87,11 @@ describe('LabReportResults component tests', () => {
                 <LabReportResults data={labReports} totalResults={1} handlePagination={() => {}} currentPage={0} />
             </BrowserRouter>
         );
```
Comment (mdavis-enquizit) [inline apps/modernization-ui/src/apps/search/event/components/labReportSearch/LabReportResults.spec.tsx:89]: no good

File: apps/modernization-ui/src/apps/search/event/components/InvestigationSearch/InvestigationChips.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { InvestigationFilter } from 'generated/graphql/schema';
 import { SearchCriteria, SearchCriteriaContext } from 'providers/SearchCriteriaContext';
-import Chip from './Chip';
+import Chip from 'apps/search/advancedSearch/components/chips/Chip';
```
Comment (mdavis-enquizit) [inline apps/modernization-ui/src/apps/search/event/components/InvestigationSearch/InvestigationChips.tsx:4]: Chip has to go somewhere?

**Review by adamloup-enquizit — CHANGES_REQUESTED** [review]
`Verify Modernization UI / Verify and Ensure (pull_request)` is failing due to a failure in the build.  The failure is occurring because the case for the `labReportSearch` folder was changed.  This is not picked up by GIT because HFS is hiding it.  You could use `git mv {old-path} {New-Path}` to ensure the change is included.  A `--force` may be needed.

Another way of doing this is to change how GIT handles case, you can do this by running `git config core.ignorecase false`.  

This PR cannot be merged if there are conflicts.  These should be addressed before asking for reviews.

**Comment by mdavis-enquizit** [comment]
It tough to switch branches because GIT struggles tell the difference in  directory name "labReportSearch" vs "LabReportSearch"
Screenshot 2024-01-09 at 3 43 13 PM

**Review by adamloup-enquizit — APPROVED** [review]

**Review by hclarkEnq — APPROVED** [review]

**Pull request merged**